### DISCUSSION TITLE
After enable varnish cache breeze theme some of the header template file is not rendering.

### DISCUSSION THREAD

**Discussion by DarshilTech** [breezefront/community#221] [1. Help]
@vovayatsyuk 
can you please help on this.

Also, that was showing as in inspect
<esi:include src="http://127.0.0.1/rb-ee/page_cache/block/esi/blocks/%5B%22catalog.topnav%22%5D/handles/WyJkZWZhdWx0IiwiY21zX2luZGV4X2luZGV4IiwiY21zX3BhZ2VfdmlldyJd/"></esi:include>

**Comment by vovayatsyuk** [comment]
It's a common issue with Magento and I doubt if it's Breeze related. You can ensure if it's a Varnish misconfiguration by changing a theme to luma.

Here is a manual on how to fix the issue: https://docs.swissuplabs.com/m2/extensions/navigationpro/troubleshooting/missing-top-navigation-when-varnish-is-used/
